Components/Config.php

```diff
@@ -517,6 +528,29 @@ private function getShopRepository()
         return $this->shopRepository;
     }
 
+    private function getPriceGateway()
+    {
+        if (!$this->priceGateway) {
+            $this->priceGateway = new \ShopwarePlugins\Connect\Components\PriceGateway(
```

Comment:
check if it's possible to inject priceGateway and customerGroupRepository in constructor